# Author: Caspar van Leeuwen
# Company: SURF
# Without this patch, we get an error
# flit_core.config.ConfigError: license field should be <class 'dict'>, not <class 'str'>
# because PEP 621 says the license must be a dict, if it is not just a file
# Probably, this happens only with sufficiently new versions of e.g. flit_core, which is
# probably why it wasn't an issue for upstream
diff -Nru mypy_extensions-1.1.0.orig/pyproject.toml mypy_extensions-1.1.0/pyproject.toml
--- mypy_extensions-1.1.0.orig/pyproject.toml	2025-06-16 16:16:04.395222990 +0200
+++ mypy_extensions-1.1.0/pyproject.toml	2025-06-16 16:24:51.865105381 +0200
@@ -10,7 +10,7 @@
 authors     = [
     {name = "The mypy developers", email = "jukka.lehtosalo@iki.fi"}
 ]
-license     = "MIT"
+license     = {text = "MIT"}
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
